dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeFactory.java

```diff
@@ -378,10 +398,12 @@ private static MultiTree dbToMultiTree(Map<String, Object> row) {
      * @param english
      * @throws DotDataException
      */
+
     @Deprecated
     public static void saveMultiTree(MultiTree multiTree, long lang) throws DotDataException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)